### PR TITLE
sourcemap for next

### PR DESCRIPTION
**Pull request by mansaj**
# References: 
Jira: MNTOR-
Figma: 

# Description

# Screenshot (if applicable)

Not applicable.

# How to test

# Checklist (Definition of Done)
- [ ] Localization strings (if needed) have been added.
- [ ] Commits in this PR are minimal and [have descriptive commit messages](https://chris.beams.io/posts/git-commit/).
- [ ] I've added or updated the relevant sections in readme and/or code comments
- [ ] I've added a unit test to test for potential regressions of this bug.
- [ ] Product Owner accepted the User Story (demo of functionality completed) or waived the privilege.
- [ ] All acceptance criteria are met.
- [ ] Jira ticket has been updated (if needed) to match changes made during the development process.
- [ ] Jira ticket has been updated (if needed) with suggestions for QA when this PR is deployed to stage.

### CODE REVIEW

File: next.config.js [hunk 1]
```diff
@@ -160,4 +160,12 @@ const sentryWebpackPluginOptions = {
   // https://github.com/getsentry/sentry-webpack-plugin#options.
 };
 
-export default withSentryConfig(nextConfig, sentryWebpackPluginOptions)
+const sentryOptions = {
+  // Upload additional client files (increases upload size)
```
Comment (Vinnl) [inline next.config.js:164]:
```suggestion
  // Upload additional client files (increases upload size)
  // https://docs.sentry.io/platforms/javascript/guides/nextjs/manual-setup/#widen-the-upload-scope
```

File: next.config.js [hunk 2]
```diff
@@ -160,4 +160,12 @@ const sentryWebpackPluginOptions = {
...
+  widenClientFileUpload: true,
+
+  // Hides source maps from generated client bundles
+  hideSourceMaps: true,
```
Comment (Vinnl) [inline next.config.js:168]: I don't think we need to hide these, and as an open source org, I actually like making our source code more accessible :)

```suggestion
```

**Review by Vinnl — APPROVED** [review]
Not sure if this actually helps (docs make it sound like it'll mostly upload Next.js source code), but can't harm to try.

**Comment by mansaj** [comment]
> Not sure if this actually helps (docs make it sound like it'll mostly upload Next.js source code), but can't harm to try.

hmm good call.. maybe are just uploading Next.js part of the map and missing the old code

File: next.config.js (hunk 2 above)
Comment (rhelmer) [reply next.config.js:169]: Yeah agree, no point in hiding, the source is open anyway. Having stuff minified for perf reasons is great but no obfuscation intended :)

Comment (mansaj) [reply next.config.js]: So I think it's actually hidden by default (true by default). Maybe we should set it as false

**Pull request merged**